task/opm-render-bundles/0.1/opm-render-bundles.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: opm-render-bundles
+spec:
+  description: Create a catalog index and render the provided bundles into it
+  params:
+    - name: binary-image
+      description: Image in which to build catalog
+      default: registry.redhat.io/openshift4/ose-operator-registry:latest
+    - name: bundle-image
+      description: Comma separated list of bundles to add
+    - name: operator-name
+      description: Name of the Operator
+    - name: operator-version
+      description: Version of the Operator
+    - name: default-channel
+      description: The channel that subscriptions will default to if unspecified
+      default: stable
+  workspaces:
+    - name: source
+      description: Workspace with the source code
+  steps:
+    - name: opm-render-bundles
+      image: "registry.redhat.io/openshift4/ose-operator-registry:latest"
+      workingDir: $(workspaces.source.path)/source
+      securityContext:
+        runAsUser: 0
+        capabilities:
+          add:
+            - SETFCAP
+      script: |
+        #!/usr/bin/env bash
+
+        set -xe
+
+        mkdir -p catalog/
+
+        opm generate dockerfile catalog/ --binary-image=$(params.binary-image)
+
+        opm init $(params.operator-name) \
+          --default-channel=$(params.default-channel) \
+          --description=./README.md \
+          --output=yaml > catalog/operator.yaml
+
+        opm render $(params.bundle-image) --output=yaml >> catalog/operator.yaml
+
+        cat << EOF >> catalog/operator.yaml
+        ---
+        schema: olm.channel
+        package: $(params.operator-name)
+        name: $(params.default-channel)
+        entries:
+          - name: $(params.operator-name).$(params.operator-version)
+            skipRange: ">=0.0.1 <$(params.operator-version)"
+        EOF
```

Comment:
We had STONEBLD-683 done already. It is recommended to follow the convention here as well, that is to define environment variables instead of using tekton varialbe substitution directly inside script block.